### ISSUE TITLE
Ctrl+Q behaves like "close window", not "quit"

### ISSUE THREAD

**Issue by ghost** [qTox/qTox#1723]
The Ctrl+Q and Alt+F4 behaviour was discussed at #1466. [This comment](https://github.com/tux3/qTox/issues/1466#issuecomment-86700399) mentions the usual behaviour of both shortcuts, but right now Ctrl+Q closes the window, it doesn't quit qTox.

**Comment by ghost** [comment]
On the same "close window/application" subject: clicking the tray icon (Linux Mint Cinnamon) minimizes qTox if it's already displayed. Normally, the behaviour is to close the window on programs configured to close to the tray. Do you prefer that I open a new issue for that?

**Comment by linux-modder** [comment]
@kehugter,   on 17.1 mint by chance?  If so there is a new default in mint  that closes to tray FOR most if not all apps.

**Comment by ghost** [comment]
Yes, 17.1, but I don't understand your point, @linux-modder . I just updated to see if the new feature was released these last days and I missed it, but nothing changed. I have at least one qt4 and one qt5 application that close to tray when the tray icon is clicked, so I guess this is about qTox and not Cinnamon.

**Comment by tux3** [comment]
CTRL+Q now properly exits, even if there are calls or other windows active.

For minimizing/closing to tray, there are options for that in the settings.

**Comment by zetok** [comment]
@tux3 still doesn't behave as it should on 4eab3dcbb866fac7c7dc22386e804c9187501a7f if `Close to tray` is checked. I agree with @ProMcTagonist that `Ctrl + q` should quit the application, rather than close it to tray, which is job of exit button or its equivalent.

**Comment by ghost** [comment]
> On the same "close window/application" subject: clicking the tray icon (Linux Mint Cinnamon) minimizes qTox if it's already displayed. Normally, the behaviour is to close the window on programs configured to close to the tray. Do you prefer that I open a new issue for that?

~~For the record: now it behaves as it "should" - closing, not minimising.~~ Nope, I just forgot to uncheck "Minimize to tray".

`Ctrl + q` still closes to tray.
